target/linux/ramips/dts/mt7620n_netgear_pr2000.dts

```diff
@@ -0,0 +1,135 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620n.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "netgear,pr2000", "ralink,mt7620n-soc";
+	model = "Netgear PR2000";
+
+	leds {
+		compatible = "gpio-leds";
+
+		internet {
+			label = "green:internet";
+			gpios = <&gpio3 0 GPIO_ACTIVE_LOW>;
+		};
+
+		wifi {
+			label = "green:wifi";
+			gpios = <&gpio2 4 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		usb {
+			label = "green:usb";
+			gpios = <&gpio0 0 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "usbport";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 1 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		wireless {
+			label = "wireless";
+			gpios = <&gpio0 20 GPIO_ACTIVE_LOW>;
+		};
+
+		wired {
+			label = "wired";
+			gpios = <&gpio0 21 GPIO_ACTIVE_LOW>;
+		};
+	};
+};
+
+&gpio2 {
+	status = "okay";
+};
+
+&gpio3 {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <80000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			partition@40000 {
+				compatible = "denx,uimage";
+				label = "firmware";
+				reg = <0x40000 0xf20000>;
+			};
+
+			factory: partition@f60000 {
+				label = "factory";
+				reg = <0xf60000 0x10000>;
+				read-only;
+			};
+
+			partition@f70000 {
+				label = "board_data";
+				reg = <0xf70000 0x10000>;
```

Comment:
What about 0xf80000 - 0xfb0000?

Reply:
empty

Reply:
Then please add a comment about it and don't just hide my question ...

Reply:
already added?

Reply:
So, now we have partitions there, although you said this was empty?

Reply:
yes

Reply:
So, why did you add partitions here although you said this was empty? .......................................

Reply:
Does openwrt policy require or forbid adding partitions for empty ones?

Reply:
Why can't you just simply answer my question?

Reply:
> because it may be helpful for someone else

DIdn't I answer your question???